src/functions-reference/real-valued_basic_functions.Rmd

```diff
@@ -687,12 +687,20 @@ used in the   data, transformed data, or generated quantities blocks.*
 
 ### Absolute value functions
 
+<!-- T; abs; (T x); -->
+\index{{\tt \bfseries abs }!{\tt (T x): T}|hyperpage}
+
+`T` **`abs`**`(T x)`<br>\newline
+The absolute value of x. This function works elementwise over containers such as
```

Comment:
This is also confusing because this signature only applies to real or integer values or real- or integer-valued containers.  When the argument is `complex`, it's a different type.  So we need to say that.